test/unit/base.test.js

```diff
@@ -393,6 +394,23 @@ describe('CONVERT FUNCTION TESTS ', function() {
         done();
       });
     });
+    it('#GITHUB-10229 should generate correct example is out of the schema and is falsy' +
+    issue10229, function(done) {
+      var openapi = fs.readFileSync(issue10229, 'utf8');
+      Converter.convert({ type: 'string', data: openapi }, { requestParametersResolution: 'Example' },
+        (err, conversionResult) => {
+          expect(err).to.be.null;
+          expect(conversionResult.result).to.equal(true);
+          expect(conversionResult.output.length).to.equal(1);
+          expect(conversionResult.output[0].type).to.equal('collection');
+          expect(conversionResult.output[0].data).to.have.property('info');
+          expect(conversionResult.output[0].data).to.have.property('item');
+          expect(conversionResult.output[0].data.item[0].item[0].request.url.query[0].value).to.equal('0');
+          expect(conversionResult.output[0].data.item[0].item[0].request.url.query[1].value).to.equal('');
+          expect(conversionResult.output[0].data.item[0].item[0].request.url.query[2].value).to.equal('false');
```

Comment:
@LuisTejedaS We can also add test where schema has type `null` (for openapi 3.1) and example should have that value.

Reply:
Added in latest commits